.github/json_output/action.yml

```diff
@@ -0,0 +1,31 @@
+name: 'Assert JSON output'
```

Comment:
Is this actions copied from somewhere? If we're modifying it I think we can change the name and author. This looks like just 2 bash calls using `jq` which is pretty generic. Maybe naming it something like `parse performance JSON output`

Reply:
Updated the names